src/components/Landing.js

```diff
@@ -0,0 +1,272 @@
+    referrerId &&
+      localStorageMgr.setItem(
+        STORAGE_REFERRAL_DATA_REFERRING_CHANNEL,
+        referrerId
+      )
+  }, [])
+  // store user referral
+  useEffect(() => {
+    const userReferrerId = getUrlParameterValue('u')
+    if (userReferrerId !== null && userReferrerId !== undefined) {
+      localStorageMgr.setItem(
+        STORAGE_REFERRAL_DATA_REFERRING_USER,
+        userReferrerId
+      )
+    }
+  }, [])
+
+  // function to be used in install button
+  // eslint-disable-next-line no-unused-vars
+  const onBeforeInstall = () => {
+    localStorageMgr.setItem(STORAGE_NEW_USER_IS_TAB_V4_BETA, 'true')
+    localStorageMgr.setItem(STORAGE_NEW_USER_CAUSE_ID, causeId)
+  }
+  const absolutePageURL = getAbsoluteURL(location.pathname)
+  let headerImg
+  switch (causeId) {
+    case STORAGE_SEAS_CAUSE_ID:
+      headerImg = seasHeaderImg
+      break
+    case STORAGE_CATS_CAUSE_ID:
+      headerImg = catsHeaderImg
+      break
+  }
+  return (
+    <div>
+      <HeadTags
+        title={headTitle}
+        titleTemplate={headTitleTemplate}
+        ogTitle={headOgTitle}
+        ogDescription={headOgDescription}
+        ogImage={headOgImgURLAbsolute}
+        keywords={headKeywords}
+        pageURL={absolutePageURL}
+      />
+      <Helmet>
+        <link rel="canonical" href={canonicalURL} />
+      </Helmet>
+      <AppBar color="primary" position="sticky">
+        <Toolbar>
+          <div className={cx.logoContainer}>
+            <div
+              data-test-id="logo-container"
+              style={{
+                display: 'flex',
+                justifyContent: 'flex-start',
+                alignItems: 'center',
+              }}
+            >
+              <Link to={homeURL}>
+                <img
+                  data-test-id="tab-logo-with-text"
+                  src={logoWhite}
+                  style={{ height: 40 }}
+                />
+              </Link>
+            </div>
+          </div>
+
+          <MoneyRaisedDisplay color="inherit" />
+        </Toolbar>
+      </AppBar>
+      <div className={cx.titleSection}>
+        <div className={cx.halfScreenLeft} data-test-id="title-wrapper">
+          <Typography variant="h1" color="primary">
+            {title}
+          </Typography>
+          <Typography className={cx.subtitle}>{subtitle}</Typography>
+          <InstallButton
+            className={cx.buttonStyles}
+            color="secondary"
+            size="medium"
+            onBeforeInstall={() => {
+              localStorageMgr.setItem(STORAGE_NEW_USER_IS_TAB_V4_BETA, 'true')
+              localStorageMgr.setItem(STORAGE_NEW_USER_CAUSE_ID, causeId)
+            }}
+            onUnsupportedBrowserInstallClick={() => {
+              setShowUnsupportedBrowserMessage(true)
+            }}
+          />
+        </div>
+        <div className={cx.halfScreenRight}>
+          <img src={headerImg}></img>
+        </div>
+        <div className={cx.wave}>
+          <Wave />
```

Comment:
let's make the wave component take a hex value as a prop and generate the svg fill color based on the prop